### DISCUSSION TITLE
Move .wingetui from %USERPROFILE%

### DISCUSSION THREAD

**Discussion by tony-sol** [Devolutions/UniGetUI#2076] [Q&A] [answered]
I want to keep my HOME dir as much clean, as possible? so here is the question - is there a way to move `.wingetui` directory (along with `.chocolatey`) from `%USERPROFILE%` (`C:\Users\%user%\`)?

Maybe there is a setting to move this to `%APPDATA%` or another win-specific location?

**Comment by marticliment** [comment, marked as answer]
Please see https://github.com/marticliment/WingetUI/issues/907
